x/pairing/client/cli/tx_modify_provider.go

```diff
@@ -190,6 +191,11 @@ func CmdModifyProvider() *cobra.Command {
 				providerEntry.DelegateLimit,
 				providerEntry.DelegateCommission,
 			)
+
+			if msg.DelegateLimit.Denom != "ulava" {
```

Comment:
So you can add it in `expected_keepers`.
I'm nit picking so we'll use as few hardcoded constants as possible, because it's hard to manage

Reply:
you can remove this check, it is done anyways in the node side

Reply:
@Yaroms Don't you think we should check that before we send it to the node?

Reply:
hmm we can do it but this will require you to do a query on the CLI side.
i think what basic cosmos tx do is check it on the node side only, so it can also be our standard